tests/unit/services/design.service.spec.ts

```diff
@@ -0,0 +1,219 @@
+    const { classList } = eksternLink!;
+    expect(classList.contains('button')).toEqual(false);
+    expect(classList.contains('button-primary')).toEqual(false);
+  });
+
+  it('knap "næste" vil være deaktiveret indtil man har valgt en svarmulighed', () => {
+    // ARRANGE
+    const buttonNextSelector = '#button-next';
+    const appContainer = document.createElement('div');
+    appContainer.classList.add(APP_CONTAINER_CLASS);
+
+    const navigationElement = document.createElement('div');
+    navigationElement.classList.add('gm-navigation');
+
+    const svarmulighedElement = document.createElement('div');
+    svarmulighedElement.classList.add('gm-svarmuligheder');
+
+    const svarElement = document.createElement('input');
+    svarElement.type = 'radio';
+    svarElement.classList.add('gm-form-radio');
+
+    const svarElement2 = document.createElement('input');
+    svarElement2.type = 'radio';
+    svarElement2.id = 'input-radio';
+    svarElement2.classList.add('gm-form-radio');
+
+    const buttonElement = document.createElement('button');
+    buttonElement.type = 'button';
+    buttonElement.id = 'button-next';
+    buttonElement.classList.add('gm-knap');
+    buttonElement.classList.add('gm-primary');
+
+    svarmulighedElement.appendChild(svarElement);
+    svarmulighedElement.appendChild(svarElement2);
+    navigationElement.appendChild(buttonElement);
+    appContainer.appendChild(navigationElement);
+    appContainer.appendChild(svarmulighedElement);
+    document.body.appendChild(appContainer);
+
+    designService.applyDesign();
+    expect(document.querySelector(buttonNextSelector)?.getAttribute('disabled')).toEqual('disabled');
+
+    // ACT
+    (document.querySelector('#input-radio') as HTMLButtonElement).click();
+    designService.applyDesign();
+
+    // ASSERT
+    expect(document.querySelector(buttonNextSelector)?.getAttribute('disabled')).toEqual(null);
+  });
+
+  it.each([
+    {
+      element: 'h2',
+      embedded: true,
+      description: 'køres som dynamisk indholdselment'
+    },
+    {
+      element: 'h1',
+      embedded: false,
+      description: 'køres som selvstændingt indholdselement'
+    }
+  ])('kan ændre overskrift til $element når applikation $description', ({ embedded, element, description }) => {
+    // ARRANGE
+    document.body.innerHTML = '';
+    const appContainer = document.createElement('div');
+    appContainer.classList.add(APP_CONTAINER_CLASS);
+
+    const overskriftElement = document.createElement('h3');
+    overskriftElement.classList.add('gm-overskrift');
+
+    appContainer.appendChild(overskriftElement);
+    document.body.appendChild(appContainer);
+
+    // ACT
+    designService.applyDesign(embedded);
+
+    // ASSERT
+    if (embedded) {
+      const h2 = document.querySelector(element);
+      expect(h2).toBeDefined();
+    } else {
+      const h1 = document.querySelector(element);
+      expect(h1).toBeDefined();
+    }
```

Comment:
Du gør stort set det samme her. Kunne du ikke nøjes med: 

```
const headerElement = document.querySelector(element);
expect(headerElement).toBeDefined();
```

Så slipper du helt for at angive `embedded`